activiti-cloud-service-common/activiti-cloud-service-messaging-config/src/main/java/org/activiti/cloud/common/messaging/config/ConnectorConfiguration.java

```diff
@@ -99,12 +114,15 @@ public Object postProcessAfterInitialization(Object bean, String beanName) throw
                             IntegrationFlow connectorFlow = IntegrationFlow
                                 .from(
                                     getGatewayInterface(Function.class.isInstance(bean)),
-                                    gateway -> gateway.replyTimeout(0L).errorChannel("errorChannel")
+                                    gateway -> gateway.replyTimeout(0L)
                                 )
                                 .log(LoggingHandler.Level.DEBUG, beanName + ".integrationRequest")
                                 .filter(
                                     selector,
-                                    filter -> filter.discardChannel("nullChannel").throwExceptionOnRejection(true)
+                                    filter ->
+                                        filter
+                                            .discardChannel(CONNECTOR_BINDING_SELECTOR_DISCARD_CHANNEL)
+                                            .throwExceptionOnRejection(false)
```

Comment:
Was "true" changed to "false" on purpose here please? (Reading other changes, I guess so, but I was curious to why)

Reply:
The selector is used to apply conditional SpEL expressions to incoming messages. It was unnecessary to throw exception on conditional filter rejection since the exception was propagating to the error channel and triggering error handling plugin. It is better to sent discarded message to a dedicated channel to be able to use debug level for logging when needed.